bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/async/BulkMutation.java

```diff
@@ -71,7 +74,7 @@ public BulkMutation(String tableName) {
   }
 
   public boolean isFull() {
-    return size >= MAX_SIZE || futures.size() >= MAX_COUNT;
+    return size >= maxRpcSize || futures.size() >= maxRpcCount;
```

Comment:
I'm tempted to say that the fullness logic dont fully belong in the BulkMutation. There is nothing preventing consumers of this class to keep adding mutations even after fullness. Maybe the class should just report a rpc size and a rpc count, and the consumer should check the fullness and decide what to do?